matching/test/imp.kore

```diff
@@ -0,0 +1,2627 @@
+[]
```

Comment:
can we delete this and point it at the same test/imp.kore in the root?

Reply:
Deleted, it isn't actually being used right now by the haste test anyway.